x-pack/plugins/ux/public/services/data/get_es_filter.ts

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Might be nice to add some basic unit tests for this file.

Reply:
https://github.com/elastic/kibana/pull/132548/commits/a0477c5a35f2261e2186c0b3ca80a389aa5ac067